media-libs/gst_libav/gst_libav-1.26.2.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="GStreamer FFmpeg plugin for libav"
+DESCRIPTION="This GStreamer FFmpeg libav plugin supports a large number of audio \
+and video compression formats through the use of the FFmpeg library. "
+HOMEPAGE="https://gstreamer.freedesktop.org/"
+COPYRIGHT="Ronald Bultje"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://gstreamer.freedesktop.org/src/gst-libav/gst-libav-$portVersion.tar.xz"
+CHECKSUM_SHA256="2eceba9cae4c495bb4ea134c27f010356036f1fa1972db5f54833f5f6c9f8db0"
+SOURCE_DIR="gst-libav-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gst_libav$secondaryArchSuffix = $portVersion
+	lib:libgstlibav$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavfilter$secondaryArchSuffix
+	lib:libgirepository_1.0$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgstapp_1.0$secondaryArchSuffix
+	lib:libgstaudio_1.0$secondaryArchSuffix
+	lib:libgstbase_1.0$secondaryArchSuffix
+	lib:libgstreamer_1.0$secondaryArchSuffix
+	lib:libgstvideo_1.0$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+SUPPLEMENTS="
+	gstreamer$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libavcodec$secondaryArchSuffix >= 60
+	devel:libavfilter$secondaryArchSuffix >= 9
+	devel:libgirepository_1.0$secondaryArchSuffix  >= 1.0.0
+	devel:libglib_2.0$secondaryArchSuffix >= 0.7800.0
+	devel:libgstapp_1.0$secondaryArchSuffix >= 0.2407.0
+	devel:libgstaudio_1.0$secondaryArchSuffix >= 0.2407.0
+	devel:libgstreamer_1.0$secondaryArchSuffix >= 0.2407.0
+	devel:libgstvideo_1.0$secondaryArchSuffix >= 0.2407.0
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:bison
+	cmd:cmake
```

Comment:
This can probably be cleaned up. Please only list commands commands that are actually required. I doubt that e.g. cmake and make are required when this uses meson and ninja.

Reply:
+1, although cmake is sometimes used in a way pkg-config files are sniffed. (didn't check but probably can be dropped)

Reply:
If cmake is needed or useful, then it should be kept of course, but there are a few other dependencies also in this list which are questionable.

Reply:
Correct, focusing here atm at gstreamer where a few things should be changed also. :)

Reply:
Resolved